src/midcom/datamanager/datamanager.php

```diff
@@ -219,4 +219,9 @@ public function get_content_html()
         }
         return $ret;
     }
+
+    public function get_schemadb()
```

Comment:
This function doesn't seem to be used. Is there any particular reason you added it?